src/Insights/Microsoft.Azure.Insights.nuspec

```diff
@@ -21,7 +21,10 @@
       </group>
     </references>
     <dependencies>
+      <dependency id="Hyak.Common" version="1.0.1" />
       <dependency id="Microsoft.Azure.Common" version="[2.0.1,3.0)" />
+      <dependency id="Microsoft.WindowsAzure.Storage" version="4.0.0" />
+      <dependency id="Newtonsoft.Json" version="6.0.4" />
```

Comment:
Remove this dependency, The Newtonsoft.Json version comes from your dependencies